dts/xtensa/espressif/esp32.dtsi

```diff
@@ -31,11 +31,27 @@
 	};
 
 	soc {
-		uart0: uart@40008fd0 {
+		uart0: uart@3ff40000 {
 			compatible = "espressif,esp32-uart";
-			reg = <0x40008fd0 0x400>;
-			label = "ROMUART";
+			reg = <0x3ff40000 0x400>;
+			//interrupts = <12>;
```

Comment:
Is commenting out the `interrupts` intentional? If this is intentional, maybe add a comment on why.

Also, our coding guideline says to use `/* */` instead of `//` for comments.

Reply:
> Is commenting out the `interrupts` intentional? If this is intentional, maybe add a comment on why.

yes, because esp32 interrupt controller still not supported in device tree.
as a temp solution there is 3 macros in `uart_esp32.c` for IRQ numbers.

> Also, our coding guideline says to use `/* */` instead of `//` for comments.

dunno what is the best option here:
1. implement a dummy interrupt-controller node for esp32 then remove `//`
2. change this line to 
```
/* interrupts = <12>; - TODO: Enable interrupts when interrupt-controller got supported in device tree*/
```
3. remove this line without leaving any notes/comments

Reply:
One more thing
do we need a copyright notice for `dts/bindings/serial/espressif,esp32-uart.yaml` ?
if yes should it be me or the one who created the file ?

Reply:
TODO (or better FIXME) would be fine. It acts as a reminder that we need to do something about it.

Not sure about the copyright. Since we have other yaml files without copyright, maybe just leave it as-is, for now? We can always go back with git log and figure it out later.